code/src/reducers/tasks.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable max-len */
+import { createSlice } from '@reduxjs/toolkit';
+
+// The 'todo slice' has an initial state with an array of one todo item. Each todo item has an 'id', 'text', and 'isDone' property.
```

Comment:
I like that you kept comments here with explanations for each thing in the code - it makes it much easier to follow and understand it.
Good job!